languages/tgmpa-vi_VN.po

```diff
@@ -15,7 +17,7 @@ msgstr ""
 "X-Poedit-KeywordsList: __;_e;_n:1,2;_x:1,2c;_ex:1,2c;_nx:4c,1,2;esc_attr__;"
 "esc_attr_e;esc_attr_x:1,2c;esc_html__;esc_html_e;esc_html_x:1,2c;_n_noop:1,2;"
 "_nx_noop:3c,1,2;__ngettext_noop:1,2\n"
-"Last-Translator: \n"
+"Last-Translator: BBLand.NET\n"
```

Comment:
The expectation here is the name and email address of the individual, not the team. See https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html